pysat/tests/test_utils_io.py

```diff
@@ -1706,36 +1689,12 @@ def test_remove_netcdf4_standards(self, caplog):
 class TestMetaTranslationXarray(TestMetaTranslation):
     """Test meta translation when writing/loading files xarray Instrument."""
 
-    def setup_method(self):
-        """Create test environment."""
-
-        self.test_inst = pysat.Instrument('pysat', 'testing_xarray',
-                                          num_samples=5, use_header=True)
-        self.test_date = pysat.instruments.pysat_testing_xarray._test_dates
-        self.test_date = self.test_date['']['']
-        self.test_inst.load(date=self.test_date)
-        self.meta_dict = self.test_inst.meta.to_dict()
-        self.out = None
-
-        return
-
-    def teardown_method(self):
-        """Cleanup test environment."""
-
-        del self.test_inst, self.test_date, self.out, self.meta_dict
-
-        return
-
-
-class TestMetaTranslation2DXarray(TestMetaTranslation):
-    """Test meta translation when writing/loading files xarray2d Instrument."""
-
     def setup_method(self):
         """Create test environment."""
 
         self.test_inst = pysat.Instrument('pysat', 'ndtesting',
                                           num_samples=5, use_header=True)
-        self.test_date = pysat.instruments.pysat_testing_xarray._test_dates
+        self.test_date = pysat.instruments.pysat_ndtesting._test_dates
         self.test_date = self.test_date['']['']
```

Comment:
```suggestion
        self.test_date = pysat.instruments.pysat_ndtesting._test_dates['']['']
```
Can break on `[` if line is too long.